lib/hammer_cli_foreman/hostgroup.rb

```diff
@@ -190,6 +190,15 @@ def validate_options
       end
     end
 
+    class RebuildConfigCommand < HammerCLIForeman::SingleResourceCommand
+      action :rebuild_config
+      command_name "rebuild-config"
+      desc _('Rebuild orchestration related configurations for hostgroup')
```

Comment:
We prefer to keep as much descriptions from API docs as possible. Is this description providing any extra value over the original one ('Rebuild orchestration config')?

Reply:
Probably not, honestly I just copied the code from `host.rb`.

Reply:
Removed `desc`.